src/components/github-contributors-list/github-contributors-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { GithubContributorResponse } from './github-contributors-types';
+
+export async function fetchContributors(organizationName: string, packageName: string, page: number) {
+  try {
+    const request = await fetch(
+      `https://api.github.com/repos/${organizationName}/${packageName}/contributors?per_page=100&page=${page}&order=desc`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    );
+
+    const contributors: GithubContributorResponse[] = await request.json();
+    return contributors;
```

Comment:
Thought that in case the response type isn't `GithubContributorResponse[]` it would throw an error here, which will be catched by `catch` statement. No?

Reply:
In runtime?

Reply:
Yeah

Reply:
I don't think it can throw type errors in runtime

Reply:
You are right
So like this it's better?
![image](https://github.com/mondaycom/vibe-storybook-components/assets/104433616/36d980b0-2c19-4753-9dd2-6b9c1bf7f95d)